pymobiledevice3/cli/remote.py

```diff
@@ -92,17 +93,32 @@ async def start_quic_tunnel(service_provider: RemoteServiceDiscoveryService, sec
 
 
 @remote_cli.command('start-quic-tunnel')
+@click.option('--udid', help='UDID for a specific device to look for')
 @click.option('--secrets', type=click.File('wt'), help='TLS keyfile for decrypting with Wireshark')
-def cli_start_quic_tunnel(secrets: TextIO):
+def cli_start_quic_tunnel(udid: str, secrets: TextIO):
     """ start quic tunnel """
     devices = get_device_list()
     if not devices:
-        print('No device could be found')
-        return
+        # no devices were found
+        raise NoDeviceConnectedError()
     if len(devices) == 1:
+        # only one device found
         rsd = devices[0]
     else:
-        rsd = prompt_device_list(devices)
+        # several devices were found
+        if udid is not None:
+            # show prompt if non explicitly selected
```

Comment:
Question: Should this be udid is None? If you don't specify a udid on the command line then udid will be None in which case we want it to promtp right?

Reply:
nice catch! is was supposed to be the opposite